corehq/apps/hqwebapp/middleware.py

```diff
@@ -27,7 +27,7 @@ def _reject(self, request, reason):
         _assert = soft_assert('{}@{}'.format('sreddy+logs', 'dimagi.com'), exponential_backoff=False)
         _assert(False, warning)
 
-        if request_url.find('login'):
+        if '/login/' in request_url:
```

Comment:
Don't like this check. It's possibly too permissive. The previous check (if it had been phrased more correctly as something like `'login' in request_url`) is trivially exploited by creating a domain with the letters 'login' embedded somewhere in the name. I think that's no longer possible, but I'm not confident that I can imagine all similar attack vectors to know that the new check is completely safe either. Would be better to do a positive match on known login URL patterns, but I'm not sure how to do that easily.

Why is it important that we disable CSRF protection on login URLs?